io/disk/softwareraid.py.data/Readme

```diff
@@ -12,6 +12,8 @@ Values to be passed in yaml file:
 
 * spare disks (space separated disks) [optional parameter]
 
+* required disks - This is a hardcoded value of minimum required disks for each raid level.
```

Comment:
@harish-24 As a yaml parameter, what you say is right. But the minimum required disks for each raid level is defined by mdadm, and is static. So, the confusion here. With this information, does your suggestion change ? I will go with your suggestion.

Reply:
@narasimhan-v Is there a way to find that based on the raid level and make that as the default? if that is not possible, I would suggest to document this information as well so that it provides better clarity.

Reply:
Sure, will document that.
The reason it is configurable is it changes depends on `--force` option.